test_othermlpackages/conda-pytorch.yaml

```diff
@@ -0,0 +1,15 @@
+name: fairlearn-pytorch
+channels:
+- defaults
+- pytorch
```

Comment:
This file failed to install for me, complaining about skorch. Does the conda-forge channel need to be included here?

Reply:
I added conda-forge and cpuonly to the conda file. So Pytorch should disable cuda by default

Reply:
Thanks!